### ISSUE TITLE
Android build failing `@Override`

### ISSUE THREAD

**Issue by ShMcK** [urbanairship/react-native-module#34]
RN: 0.48.1
OS: Android

Build fails with the following error:

```
ReactAirshipPackage.java:25

error: "Method does not override method from its superclass"
@Override
```

In the editor, the linter also indicates the same issue and error. 

The build succeeds if you comment out lines 25-28.

```
public class ReactAirshipPackage implements ReactPackage {

//    @Override
//    public List<Class<? extends JavaScriptModule>> createJSModules() {
//        return Collections.emptyList();
//    }

}
```

The issue seems to be related to there no longer being a createJSModules with no params.

**Comment by ShMcK** [comment]
Just noticed, issue fixed in #33 .

**Comment by rlepinski** [comment]
Fixed in 1.2.1
